sections/sprint-1/experiment.tex

```diff
@@ -104,11 +110,15 @@ \subsubsection{Precision with 5 tag}
 The $y$ coordinate fluctuated more but was still less than 25 cm.
 
 \subsection{Possible influences on the test}
-One thing that could have affected the tags is that a blackboard was used as a measure for positions. 
-As there is metal in the blackboard this could have affected the precision on the tags.
+One thing that could have affected the tags is that a whiteboard was used as a measure for positions. 
+As there is metal in the whiteboard this could have affected the precision on the tags.
 Metals are conductors, which can lead to the signal having less power and reduced range, and the signal might spend extra time trying to get through the metal.
 Since Pozyx positioning relies on calculating the time of flight, having the signal spend extra time traveling reduces accuracy \cite{pozyx-UWBObstacles}. 
 \\\\
+If we look at the $z$ coordinate in the test experiment data, it can be seen that $z$ fluctuates a lot more than the $x$ and $y$ coordinate. 
```

Comment:
```suggestion
If we look at the  $z$ coordinate in the test experiment data, it can be seen that $z$ fluctuates a lot more than the $x$ and $y$ coordinates. 
```